django_celery_beat/models.py

```diff
@@ -545,9 +553,20 @@ def save(self, *args, **kwargs):
         self.headers = self.headers or None
         if not self.enabled:
             self.last_run_at = None
+        self._clean_expires()
         self.validate_unique()
         super(PeriodicTask, self).save(*args, **kwargs)
 
+    def _clean_expires(self):
+        if self.expire_seconds and self.expires:
```

Comment:
What if expire_seconds is 0?  Both here and the expires_ property, that would throw off the logic right, or am I reading it wrong?  It looks like you shoudl be checkin "is None"?

Reply:
Thanks for your attention, it should be checked with `is None`.


In expires_ property,  `return self.expires or self.expire_seconds` is ok, because   `expires` is in the left of or expression.